docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+  babyagi:
+    build: ./
+    volumes:
+      - "./:/app"
+    profiles: ["exclude-from-up"] # Use `docker-compose run babyagi` to get an attached container
```

Comment:
Suggest "babyagi" would be be a better naming

Reply:
this profile is intended to prevent the container from being included in `docker-compose up`, renaming it might make this less intention revealing?

Reply:
Yeah, I understand why you did this. But I have some notes though:
1. There is only one container, so why do you care if people use `docker-compose up`?
2. This container can be run with `docker-compose --profile   exclude-from-up up` command. Seems a little weird

But this is just my opinion, you can totally disagree with me :)

Reply:
I don't understand much about docker, but why would we want to prevent compose up?

Reply:
I believe, that it is an adaptation of [autoGPT](https://github.com/Torantulino/Auto-GPT/blob/master/docker-compose.yml) repo. The question why they wanted to prevent `compose up` is a mystery for me.
Anyway, it's doesn't harm anyone so it could be left as is.

Reply:
The purpose in autogpt was because the entrypoint is a TTY, so up'ing the container isn't the right move since you can't easily attach to it. I suppose at the moment if babyagi doesn't allow user input at the terminal then this is not necessary and you can just up the container. I suspect in the near future this will change and this setup will be necessary but it's a fair point that it's not necessary atm